From 4236d0f43589c8fa9f89f9c7a1f922d6c4d42320 Mon Sep 17 00:00:00 2001
From: Jason Hung <jhung@globalscaletechnologies.com>
Date: Tue, 26 Nov 2024 15:25:28 +0800
Subject: [PATCH] dts: gti-mt510ss-evk: fix uart2 and spi2 pinmux conflict
 issue

disable spi2 to fix uart2 and spi2 pinmux conflict issue
---
 arch/arm64/boot/dts/mediatek/gti-mt510ss-evk.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/mediatek/gti-mt510ss-evk.dts b/arch/arm64/boot/dts/mediatek/gti-mt510ss-evk.dts
index dd127e3c526c..388401c5de6e 100644
--- a/arch/arm64/boot/dts/mediatek/gti-mt510ss-evk.dts
+++ b/arch/arm64/boot/dts/mediatek/gti-mt510ss-evk.dts
@@ -1196,7 +1196,7 @@ &spi2 {
 	mediatek,pad-select = <0>;
 	#address-cells = <1>;
 	#size-cells = <0>;
-	status = "okay";
+	status = "disabled";
 
 	spidev@0 {
 		compatible = "mediatek,aiot-board";
-- 
2.34.1

